docs/source/supported/frontier.rst

```diff
@@ -1,4 +1,155 @@
-=============
-ORNL Frontier
-=============
+====================
+Frontier (OLCF/ORNL)
+====================
+
+Platform user guide
+===================
+
+https://docs.olcf.ornl.gov/systems/frontier_user_guide.html
+
+General description
+===================
+
+* Resource manager - ``SLURM``
+* Launch methods (per platform ID)
+
+  * ``ornl.frontier`` - ``SRUN``
+
+* Configuration per node (9,408 nodes in total)
+
+  * 64 CPU cores, each core has 2 threads (``SMT=2``)
+  * 8 GPUs (AMD MI250X)
+  * 512 GiB of memory
+
+.. note::
+   RADICAL-Pilot allows to use the ``--constraint`` option
```

Comment:
I tried to edit this paragraph but I do not understand it. Could you edit it further or could we discuss about it so that I can understand what we need to write?

Reply:
reworked